R/oauth-init.R

```diff
@@ -109,10 +111,12 @@ init_oauth2.0 <- function(endpoint, app, scope = NULL, user_params = NULL,
 
   if (isTRUE(use_basic_auth)) {
     req <- POST(endpoint$access, encode = "form", body = req_params,
-      authenticate(app$key, app$secret, type = "basic"))
+      authenticate(app$key, app$secret, type = "basic"),
```

Comment:
I've made the edits and pushed.  Why is RStudio `Reindent-Lines` not `tidyverse`-style-compatible?